styles/SideBar.css

```
@@ -0,0 +1,62 @@
+/* ========== Sidebar Component ========== */
+
+.top-sidebar-container span,
+h3 {
+  @apply text-white;
+}
+
+.top-sidebar-container :first-child {
+  @apply text-2xl font-semibold;
+}
+
+.top-sidebar-container {
+  @apply pt-12 ml-12 flex flex-col gap-y-2;
+}
+
+.side-menu {
+  @apply flex gap-x-3 pt-4 pb-4 pl-6 text-2xl;
+}
+
+.sidebar-container {
+  @apply flex flex-col;
+}
+
+.side-menu-icon {
+  @apply h-10 w-10 self-center justify-self-center;
+}
```

ใส่ paddding right ให้มันอีกนิ๊สสสนึง นิ๊สสนึง จริงๆ
```suggestion

.side-menu-icon {
  @apply h-10 w-10 pr-1 self-center justify-self-center;
}
```